### ISSUE TITLE
`nrn.h.load_file("./init.hoc");` crashes notebook

### ISSUE THREAD

**Issue by Anner-deJong** [BlueBrain/MOOC-neurons-and-synapses-2017#69]
When arriving at and running the cell: **`nrn.h.load_file("./init.hoc");` of the 'Week 6. NMC portal' notebook, the kernel keeps crashing**. Not sure why. (notebook taking from HBP collab, but seems the same as [github version](https://github.com/BlueBrain/MOOC-neurons-and-synapses-2017/blob/master/NMCPortal/NMCPortal.ipynb)). Tried both locally (without compiling `!nrnivmodl mechanisms`) and in a google colab (with compiling).

--------------------

1. I tried to go into `init.hoc` and comment out lines 1 by 1. Not giving me a definite pattern but it seems: calling `create_cell()` causes the crash. If `nrn.h.load_file("./init.hoc");` doesnt crash (with init.hoc line `create_cell(synapses_enabled)` commented out), the following notebook cell will (`nrn.h.create_cell(arg);`).
   - The `nrn.h.load_file("./init.hoc");` command, depending on how much is commented out, will issue an error, see below.
2. Same as with assignments of earlier weeks, the prescribed package versions give me clashes; I remove them and so far everything worked fine, but it might be that this is causing the issue? See below for versions.

--------------------
`nrn.h.load_file("./init.hoc");` error when init.hoc has everything but 4 load_file() lines commented out:
```
NEURON: Ih is not a MECHANISM
 in biophysics.hoc near line 50
 		insert Ih
           ^
        xopen("biophysics.hoc")
      execute1("{xopen("bi...")
    load_file("biophysics.hoc")
  xopen("createsimu...")
and others
NEURON: ProbGABAAB_EMS is not a template
 in synapses.hoc near line 274
             section.sec synapse = new ProbGABAAB_EMS(seg_x)
                                                           ^
        xopen("synapses.hoc")
      execute1("{xopen("sy...")
    load_file("synapses/s...")
  xopen("template.hoc")
and others
```

--------------------
package versions:
```
python: 3.7
neuron:  7.8.2
bluepyopt:  1.9.147
matplotlib:  3.2.2
numpy:  1.19.5
neurom: 1.8.0
neurom: 7.8.2
efel: 3.1.88
```

**Comment by wvangeit** [comment]
This error message probably means that for some reason the mechanisms (like Ih) / MOD files could not be loaded.
That you get it locally without running nrnivmodl mechanisms, it is normal. What was the reason you didn't run nrnivmodl?

For the run on the HBP collab. It could be that an error happened during the compilation of the MOD files. Does the notebook show some particular output after executing !nrnivmodl mechanisms? Could you copy-paste it here, thank you.

**Comment by Anner-deJong** [comment]
Thanks Werner, will get back to you.
- locally my work laptop prevents executing nrnivmodl
- remote in the google colab I confirmed before nrnivmodl compiled everything successfully. Will try and confirm this again, and paste error + detail.

**Comment by Anner-deJong** [comment]
Thanks again for confirming Werner.  

Confirmed both the `create_cell()` and `nrn.h.load_file("./init.hoc")` now run fine in my remote colab.

------------------------
Not sure what I did wrong the first time, presumably I didnt run the nrnivmodl correctly, or I messed up the environment with wrong versions too much (currently still some packages have other versions, notable neurom which wont pip3 install for 1.4.2.
